parser/src/cfg/mod.rs

```diff
@@ -251,12 +271,60 @@ fn parse_cfg_raw(
     KeySeqsToFKeys,
     Overrides,
 )> {
-    let text = std::fs::read_to_string(p).map_err(|e| miette::miette!("{e}"))?;
-    let cfg_filename = p.to_string_lossy().to_string();
-    parse_cfg_raw_string(&text, s, &cfg_filename).map_err(error_with_source)
+    const INVALID_PATH_ERROR: &str = "The provided config file path is not a valid";
```

Comment:
Not sure if this is an incomplete sentence or has an extra word.
```suggestion
    const INVALID_PATH_ERROR: &str = "The provided config file path is not a valid file";
```

```suggestion
    const INVALID_PATH_ERROR: &str = "The provided config file path is not valid";
```